src/recipe/admin/src/runtime/components/MenuPage.vue

```diff
@@ -5,23 +5,75 @@
       name="header"
       :descriptor="header"
     />
-    <div class="grid gap-4 grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 2xl:grid-cols-6">
-      <Bake
-        v-for="(link, i) in links"
-        :key="link.schema.route"
-        :name="`links/${i}`"
-        :descriptor="link"
-      />
+    <div
+      v-if="sectionsData.length > 0"
+      class="flex flex-col gap-6"
+    >
+      <div
+        v-for="section in sectionsData"
+        :key="section.title"
+      >
+        <h2
+          v-if="section.title"
+          class="
+            text-zinc-400 dark:text-zinc-600
+            uppercase text-xs font-bold
+          "
+        >
+          {{ section.title }}
+        </h2>
+        <Divider
+          v-if="section.title"
+          type="dashed"
+          class="mt-2"
+        />
+        <div class="grid gap-4 grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 2xl:grid-cols-6">
+          <Bake
+            v-for="(link, i) in section.links"
+            :key="link.component.schema.route"
```

Comment:
now that it has `title` you can use it in `:key` instead of relying on `schema` to have `route`